backstop/tests/table.html

```diff
@@ -35,6 +35,10 @@
       rel="stylesheet"
       href="../../lib/css/icon.css"
     />
+    <link
+      rel="stylesheet"
+      href="../../lib/css/text.css"
```

Comment:
What for?

Reply:
There must have been a reason, but I can't figure out what it was & tests arent showing anything.  Removed.